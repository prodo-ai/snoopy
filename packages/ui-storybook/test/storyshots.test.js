"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const addon_storyshots_1 = require("@storybook/addon-storyshots");
addon_storyshots_1.default({});
//# sourceMappingURL=storyshots.test.js.map