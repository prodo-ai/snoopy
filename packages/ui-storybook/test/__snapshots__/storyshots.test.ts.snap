// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Component base 1`] = `
<div
  className="sc-ifAKCX keGVmf"
>
  <div
    className="sc-EHOje fwuUJE"
  >
    <div
      className="sc-bdVaJa hLhmaC"
    >
      <div
        className="example-title sc-bxivhb gblRRH"
      >
        Example 1
      </div>
      <div
        className="sc-bwzfXH eqVgEE"
      >
        <div
          className="example-contents sc-htpNat gZyIoC"
        >
          <div>
            Tom
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Sidebar closed 1`] = `
<div
  className="sidebar sc-iwsKbI ftanjX"
>
  <div
    className="sc-fjdhpX gddroE"
  >
    <span
      className="sidebar-toggle sc-jzJRlG bGoKof"
      onClick={[Function]}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-list fa-w-16 fa-lg "
        data-icon="list"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M128 116V76c0-8.837 7.163-16 16-16h352c8.837 0 16 7.163 16 16v40c0 8.837-7.163 16-16 16H144c-8.837 0-16-7.163-16-16zm16 176h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zM16 144h64c8.837 0 16-7.163 16-16V64c0-8.837-7.163-16-16-16H16C7.163 48 0 55.163 0 64v64c0 8.837 7.163 16 16 16zm0 160h64c8.837 0 16-7.163 16-16v-64c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v64c0 8.837 7.163 16 16 16zm0 160h64c8.837 0 16-7.163 16-16v-64c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v64c0 8.837 7.163 16 16 16z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </span>
  </div>
</div>
`;

exports[`Storyshots Sidebar open 1`] = `
<div
  className="sidebar sc-iwsKbI gSIGHZ"
>
  <div
    className="sc-fjdhpX gddroE"
  >
    <a
      className="sc-gqjmRU eYqZJu"
      href="/"
      onClick={[Function]}
    >
      <div
        className="title sc-VigVT jFCsNF"
      >
        <img
          alt=""
          className="logo sc-dnqmqq cnjRDI"
        />
        Snoopy
      </div>
    </a>
  </div>
  <hr
    className="sc-jTzLTM jEoeNn"
  />
  <div
    className="component-list sc-bZQynM AMncx"
  >
    <a
      className="sc-htoDjs jssEWI"
      href="/App"
      onClick={[Function]}
    >
      <div
        className="sc-gzVnrw kXGagn"
        selected={false}
      >
        App
      </div>
    </a>
  </div>
</div>
`;
